card/client.go

```diff
@@ -74,8 +74,8 @@ func (c *Client) CaptureChargeWithContext(ctx context.Context, data *CaptureChar
 }
 
 // GetCharge gets a card charge
-func (c *Client) GetCharge(data *CaptureChargeParams) (*xendit.CardCharge, *xendit.Error) {
-	return c.CaptureChargeWithContext(context.Background(), data)
+func (c *Client) GetCharge(data *GetChargeParams) (*xendit.CardCharge, *xendit.Error) {
```

Comment:
why this changed?

Reply:
I think this was a bug from previously

Reply:
yeah sorry, should I separate this change into another PR?

Reply:
No I think it's fine being kept here as it's a bug fix rather (eventhough it broke the signature)

Reply:
ah okay.